test/app.e2e-spec.ts

```diff
@@ -0,0 +1,24 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { INestApplication } from '@nestjs/common';
+import * as request from 'supertest';
+import { AppModule } from './../src/app.module';
+
+describe('AppController (e2e)', () => {
```

Comment:
Add a test with sample questions and see if the XML is getting created perfectly.

Reply:
I added the test case [here](https://github.com/Samagra-Development/quml2xform/pull/1/commits/b2c92266da98450540829216b1439a53c5c9f282) to check if the newly generated form files exist for real by upon hitting the newly added route.